src/Containers/CadView.jsx

```diff
@@ -363,24 +358,21 @@ export default function CadView({
               pathPrefix + (modelPath.gitpath ? modelPath.getRepoPath() : modelPath.filepath)
             }
           />}
-        <ItemPanelButton
-          model={model}
-          element={selectedElement}
-          open={showItemPanel}
-          onClickCb={() => setShowItemPanel(!showItemPanel)}
-          topOffset={PANEL_TOP}
-          placeCutPlane={() => placeCutPlane()}
-          unSelectItem={() => unSelectItems()}
-          toggleShortCutsPanel={() => setShowShortCuts(!showShortCuts)}/>
-        <div className={showItemPanel ? classes.iconGroupOpen : classes.iconGroup}>
-          <OperationsGroup
-            placeCutPlane={() => placeCutPlane()}
-            unSelectItem={()=>unSelectItems()}
-            toggleShortCutsPanel={() => setShowShortCuts(!showShortCuts)}
-          />
+        <div className={isItemPanelOpen ?
+                        classes.operationsGroupOpen :
+                        classes.operationsGroup}>
+          {viewer &&
+           <OperationsGroup
+             viewer={viewer}
+             unSelectItem={unSelectItems}
+             itemPanel={
+               <ItemPanelControl
+                 model={model}
+                 element={selectedElement}
+                 isOpenState={isItemPanelOpenState}/>}/>}
         </div>
         <LogoDark className={classes.logo}/>
-        <div className={showItemPanel ? classes.baseGroupOpen : classes.baseGroup}>
+        <div className={isItemPanelOpen ? classes.baseGroupOpen : classes.baseGroup}>
```

Comment:
when the item panel is open two instances of base group are visible
<img width="610" alt="image" src="https://user-images.githubusercontent.com/3433606/156635659-f7369bbc-c5c6-45fe-aab6-db5955571a82.png">

Reply:
I'm not seeing this.  You still getting it?